wcivf/apps/core/models.py

```diff
@@ -18,7 +18,7 @@ class LoggedPostcode(TimeStampedModel):
     def __str__(self):
         return "{0}".format(self.postcode)
 
-def log_postcode(log_dict, blocking=True):
+def log_postcode(log_dict, blocking=False):
```

Comment:
I could obv. re-write the history of this branch if you had loads of opinions about it.